### ISSUE TITLE
Hyper-parameter

### ISSUE THREAD

**Issue by deeplearnerJHB** [Kaixhin/PlaNet#18]
Hello, thanks for your effort in transferring the Planet into pytorch. I have saw that you released the experiment results about dm-control-suit and had achieved comparable performance. However, on my machine, in some simulation environment, the training curve looks very bad, such as reacher_easy and ball_in_cup. I follow the fixed default hyper-parameter setting in your.   Whether you fine-tune the hyper-parameters setting for different environments to achieve a robust result? Thank you very much!

**Comment by Kaixhin** [comment]
The default hyperparameters should be the same for all of the DM Control Suite, except for the action repeat, where these are noted in the code and in the paper itself.

**Comment by deeplearnerJHB** [comment]
Thank you for your answer. I had found the bugs in my code that leads bad results in many environment. I also found you use the same seed for all 10 test environments and keep exploration in test phase. I'm confused with such operation. May be such operation leads the quick convergence in “reacher-easy” and huge variance in “ball_in_cup-catch”?

**Comment by deeplearnerJHB** [comment]
The original paper said that the areas show percentiles 5 to 95 over 5 seeds and 10 trajectories. 
I'm confused with that sentence.

**Comment by Kaixhin** [comment]
Ah sorry that is a bug - exploration should be disabled during the test phase. I will try and rerun the experiments (this could take a while) to see if this changes anything. I only have the resources to evaluate with one seed, so will have to keep that fixed, but you should use several if possible.

I assume that means the bounds are calculated as percentiles on 50 trajectories, but you would have to check with the original author. My code simply plots the statistics over 10 trajectories for the current experiment (1 seed).

**Comment by Kaixhin** [comment]
Here's updated results on `ball_in_cup-catch` - perhaps a bit better, but still quite unstable. I'll continue to collect results for the other domains in any case.

![newplot](https://user-images.githubusercontent.com/991891/66058268-fcdcd580-e531-11e9-8319-82bc9522b5d4.png)

**Comment by deeplearnerJHB** [comment]
I'm not sure that whether you have noticed that when in test phase, you will reconstruct a envbatch object and just set the all seeds of 10 environments to 1. In fact, you just begin at the same start point for your all test running. With the training of the model, the model sometimes may make proper planning but sometimes also may make wrong decisions. Especially, the ball-in-cup
itself is an environment with the sparse reward which may potentially make the test unstable.

**Comment by deeplearnerJHB** [comment]
You can set the test environment with different seeds. Then, during test phase, some environments may success and some may failed. The result chart may looks better.

**Comment by Kaixhin** [comment]
It's true that this environment is difficult to evaluate because of the sparse reward, but I did it first because it's one that I didn't easily match the original results with - the others worked fine for me.

Have you tried adjusting the code to see if changing the seed of the test environments affects the results? Given that the planning is based on sampling from a Gaussian distribution, there is already a difference in results over trajectories even if the starting conditions are the same, so I'm not sure why this would necessarily improve results.
